requirements-dev.txt

```diff
@@ -17,4 +17,5 @@ neo @ git+https://github.com/NeuralEnsemble/python-neo.git@master
 roiextractors==0.3.1
 ndx-events==0.2.0
 parameterized==0.8.1
-pyintan==0.3.0
\ No newline at end of file
+pyintan==0.3.0
+natsort==7.1.1
```

Comment:
I know adding `natsort` to requirements makes us that much less of a lazy importer, but it really does improve readability over `regex` and it can be used in a few other interface refactors as well.